cmd/pixterm/pixterm.go

```diff
@@ -33,7 +33,7 @@ const (
 
    ___  _____  ____
   / _ \/  _/ |/_/ /____ ______ _    Made with love by Eliuk Blau
- / ___// /_>  </ __/ -_) __/  ' \   github.com/eliukblau/pixterm
+ / ___// /_>  </ __/ -_) __/  ' \   https://github.com/eliukblau/pixterm
```

Comment:
KEEP THIS WITHOUT CHANGES

Reply:
Out of curiosity, why? Having the `https://` prefix makes it possible to click the link in terminal emulators that detect links. Most terminal emulators supporting this feature will require you to hold <kbd>Ctrl</kbd>, but it's still more convenient than having to copy-paste the URL into a Web browser.

Reply:
Because A E S T H E T I C S 😎

It is because of the number of characters in the logo, so that it could be "balanced" (personal criteria). Anyways, you don't worry. I will see if I can adjust the position of the URL including the HTTPS. 👍